source/background.js

```diff
@@ -40,9 +40,10 @@ async function handleLastModified(newLastModified) {
 
 async function updateNotificationCount() {
 	const response = await getNotificationCount();
-	const {count, interval, lastModified} = response;
+	const {count, interval, notifications, lastModified} = response;
 
-	renderCount(count);
+	const badgeText = notifications.reduce((text, notification)=> text + notification.reason + '=>' + notification.subject.title + "\n" , "");
```

Comment:
This is where the weird thing happens the size of `notifications` is different than `count`, so I get a list of only one notification but I get the number 4 above notifier-for-github's badge.